src/wing/CCPACSWing.cpp

```diff
@@ -420,49 +425,69 @@ void CCPACSWing::BuildUpperLowerShells()
     lowerShape = generatorLow.Shape();
 }
 
-
 void CCPACSWing::BuildWingWithCutouts(PNamedShape& result) const
 {
 
-
     if (NumberOfControlSurfaces(*this) == 0) {
         return;
     }
 
-
     TopoDS_Compound allFlapPrisms;
     BRep_Builder compoundBuilderFlaps;
-    compoundBuilderFlaps.MakeCompound (allFlapPrisms);
+    compoundBuilderFlaps.MakeCompound(allFlapPrisms);
 
     PNamedShape fusedBoxes;
     bool first = true;
-    for ( int i = 1; i <= GetComponentSegmentCount(); i++ ) {
-
-       const CCPACSWingComponentSegment &componentSegment = GetComponentSegment(i);
-       if (!componentSegment.GetControlSurfaces().is_initialized()) continue;
-
-       const CCPACSControlSurfaces& controlSurfs = componentSegment.GetControlSurfaces().value();
-       if (!controlSurfs.GetTrailingEdgeDevices().is_initialized()) continue;
-       const CCPACSTrailingEdgeDevices& controlSurfaceDevices = controlSurfs.GetTrailingEdgeDevices().value();
+    for (int i = 1; i <= GetComponentSegmentCount(); i++) {
 
-        for ( size_t j = controlSurfaceDevices.GetTrailingEdgeDevices().size(); j > 0 ; j-- ) {
-            CCPACSTrailingEdgeDevice& controlSurfaceDevice = *controlSurfaceDevices.GetTrailingEdgeDevices().at(j-1);
+        const CCPACSWingComponentSegment& componentSegment = GetComponentSegment(i);
+        if (!componentSegment.GetControlSurfaces().is_initialized())
+            continue;
```

Comment:
Personally, I don't put much weight onto a consistent coding style. But I am particular about always using braces in control-flow, even for one-liners like this one. It is so easy to introduce bugs later on by adding a statement before the line `continue;` and forgetting to also add braces.

```suggestion
        if (!componentSegment.GetControlSurfaces().is_initialized()) {
            continue;
        }
```